config/feed-variants/20220518-variant.json

```diff
@@ -0,0 +1,109 @@
+{
+  "max_days_since_published": 15,
+  "order_by": "relevancy_score_and_publication_date",
+  "levers": {
+    "daily_decay": {
+      "cases": [
+        [0, 1],
+        [1, 0.975],
+        [2, 0.965],
+        [3, 0.955],
+        [4, 0.945],
+        [5, 0.935],
+        [6, 0.925],
+        [7, 0.922],
+        [8, 0.919],
+        [9, 0.916],
+        [10, 0.913],
+        [11, 0.91],
+        [12, 0.907],
+        [13, 0.904],
+        [14, 0.901]
+      ],
+      "fallback": 0.9
+    },
+    "comments_count_by_those_followed": {
+      "cases": [
+        [0, 0.95],
+        [1, 0.98],
+        [2, 0.99]
+      ],
+      "fallback": 0.93
+    },
+    "comments_count": {
+      "cases": [
+        [0, 0.8],
+        [1, 0.82],
+        [2, 0.84],
+        [3, 0.86],
+        [4, 0.88],
+        [5, 0.9],
+        [6, 0.92],
+        [7, 0.94],
+        [8, 0.96],
+        [9, 0.98]
+      ],
+      "fallback": 1
+    },
+    "featured_article": { "cases": [[1, 1]], "fallback": 0.85 },
+    "following_author": {
+      "cases": [
+        [1, 1],
```

Comment:
This should be `[0, 1],`; Alternatively, you could exclude the `following_author` lever (which would improve query performance.